lib/segment/src/index/field_index/full_text_index/inverted_index.rs

```diff
@@ -203,9 +177,9 @@ impl InvertedIndex {
         } else {
             let expected_frac: f64 = postings
                 .iter()
-                .map(|posting| posting.len() as f64 / self.points_count as f64)
+                .map(|posting| posting.len() as f64 / points_count as f64)
```

Comment:
Should we ensure here that `points_count` is not `0`? Maybe we can do an early return somewhere.

Reply:
That's a brilliant finding. Because previously we was sure that `points_count` is not zero (because we checked amount of postings list). But not we don't delete anything from inverted index and may meet the situation where postings list is not empty but `points_count` is zero. Fixed using additional check for `points_count != 0`

Reply:
Thanks for adding the check!